semgrep/semgrep/cli.py

```diff
@@ -200,6 +200,13 @@ def cli() -> None:
     output.add_argument(
         "--json", action="store_true", help="Output results in JSON format."
     )
+    output.add_argument(
+        "--save-output",
```

Comment:
Perhaps it should be `--save-test-output-dir` to be more specific?

Reply:
how about --save-test-output-tar?